src/components/Hero/HubHero/index.tsx

```diff
@@ -38,55 +39,51 @@ const HubHero = ({
         }}
       />
       <Stack
-        spacing="4"
-        p={{ base: "4", lg: "8" }}
-        textAlign={{ base: "center", xl: "start" }}
-        borderRadius={{ xl: "base" }}
-        bg={{ xl: "hubHeroContentBg" }}
-        position={{ xl: "absolute" }}
-        maxW={{ xl: "sm" }}
-        top={{ xl: "50%" }}
-        transform={{ xl: "translateY(-50%)" }}
-        backdropFilter={{ xl: "auto" }}
-        backdropBlur={{ xl: "base" }}
-        wordBreak="break-word"
-        sx={{
-          "inset-inline-start": { xl: "32px" },
-        }}
+        className={cn(
+          "gap-4 p-4 lg:p-8",
+          "text-center xl:text-start",
+          "xl:rounded",
+          "xl:bg-hub-hero-content-bg",
+          "xl:absolute xl:top-1/2 xl:max-w-sm",
+          "xl:-translate-y-1/2 xl:transform",
+          "xl:backdrop-blur xl:backdrop-filter",
+          "break-words",
+          "xl:start-[32px]"
+        )}
       >
         {title ? (
-          <Text
-            as="h1"
-            size="md"
-            color="body.medium"
-            fontWeight="normal"
-            textTransform="uppercase"
-          >
+          <h1 className="text-md font-normal uppercase text-body-medium">
             {title}
-          </Text>
+          </h1>
         ) : null}
         <Stack
-          alignSelf="center"
-          spacing={{ base: "2", md: "1" }}
-          maxW="container.md"
+          // --eth-sizes-container-md: 768px ==> tailwind max-w-3xl
+          className="max-w-3xl gap-2 self-center md:gap-1"
         >
-          <Heading as={title ? "h2" : "h1"} size="2xl">
-            {header}
-          </Heading>
-          <Text size="lg">{description}</Text>
+          {/* Ref: file: src/components/Hero/HubHero/index.tsx:28 */}
+          {/* 2xl font size refers to  ["4xl", null, null, "5xl"] */}
```

Comment:
What is this ref indicating?

re: the font size comment, I don't think we need to clarify that given that the classnames are self-explanatory, or am I missing something?

Reply:
It is added with reference to the Chakra Heading component at `src/@chakra-ui/components/Heading.ts:21`
The comment is added to avoid confusion between heading with size 2xl from Chakra to ["4xl", "5xl"]  in tailwind classes

Reply:
Right, the thing is that we are moving out of Chakra and these refs and clarifications won't matter when we finish the transition given that we will not have the `size` scale anymore.

JLYK we have migrated these sizes in our globals css file. https://github.com/ethereum/ethereum-org-website/blob/dev/src/styles/global.css#L219